src/FacebookAds/Object/AbstractObject.php

```diff
@@ -45,6 +45,8 @@ public function __construct() {
   public function __get($name) {
     if (array_key_exists($name, $this->data)) {
       return $this->data[$name];
+    } elseif (in_array($name, static::getFieldsEnum()->getValues())) {
```

Comment:
not yet, but since this bug can be nasty i'll probably ship a new patch version soon